db/db.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS users (userid SERIAL PRIMARY KEY,
+                                  login VARCHAR,
+                                  elo FLOAT,
+                                  botfile VARCHAR,
+                                  botstatus VARCHAR);
+
+CREATE TABLE IF NOT EXISTS games (gameid SERIAL PRIMARY KEY,
+                                  user1 INTEGER REFERENCES users(userid),
+                                  user2 INTEGER REFERENCES users(userid),
+                                  winner INTEGER REFERENCES users(userid),
+                                  generated_at TIMESTAMP,
+                                  gamefile VARCHAR)
```

Comment:
if this is for testing stuff, maybe create the tables in it's own schema so it doesn't interfere with api / other stuff?

```
create schema if not exists rps_test;
create table if not exists rps_test.users ( ... );
...
```

Reply:
Discussed offline: I am keen on staying in sync with your database schema.